exonum-java-binding-fakes/src/main/java/com/exonum/binding/fakes/mocks/MockInteraction.java

```diff
@@ -0,0 +1,91 @@
+package com.exonum.binding.fakes.mocks;
+
+import com.google.gson.Gson;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+
+/**
+ * Helper class to make testing callbacks from native code easier. It allows simple tracking of
+ * interaction with a specific method of a mocked object and then return it (by calling
+ * getInteractions() method from native side) as a JSON string.
+ */
+public class MockInteraction {
+  private final String[] arguments;
+  private final List<String> interactions;
```

Comment:
Initialize `interactions` at the same line where you declare it?
```suggestion
  private final List<String> interactions = new ArrayList();
```